packages/node-modules-inspector/src/app/components/graph/Canvas.vue

```diff
@@ -140,9 +143,9 @@ function handleDragingScroll() {
     x = container.value!.scrollLeft + e.pageX
     y = container.value!.scrollTop + e.pageY
   })
-  useEventListener('mouseleave', () => isGrabbing.value = false)
-  useEventListener('mouseup', () => isGrabbing.value = false)
-  useEventListener('mousemove', (e) => {
+  useEventListener(container, 'mouseleave', () => isGrabbing.value = false)
+  useEventListener(container, 'mouseup', () => isGrabbing.value = false)
+  useEventListener(container, 'mousemove', (e) => {
```

Comment:
This is intentionally to listen on the document. As when you drag onto other elements the dragging should still functioning.

Reply:
The reason I made this change was that dragging the scrollbar caused position jumps. (happens randomly)
https://github.com/user-attachments/assets/fe320080-9bd3-4472-92ba-b04f8c5d511b

Adding an event listener to the container happened to resolve the issue. However, it looks like another approach is needed. 😅

Reply:
This should fix the problem. https://github.com/antfu/node-modules-inspector/pull/27/commits/fb62c4f0e1fbf4060e015ffd179ac7c6297e6e42